.github/workflows/ci.yaml

```diff
@@ -63,9 +57,24 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      
+      - name: Install required dependencies
+        run: |
+          sudo snap install yq
 
-      - name: Upgrade snapd
+      - name: Upgrade linux deps
         run: |
+          sudo apt-get update
+          
+          # install security updates
+          sudo apt-get -s dist-upgrade \
```

Comment:
@zmraul can we remove this part and move to `ubuntu-24.04` instead of `ubuntu-latest` in this CI?

Reply:
dist upgrade will take us from 22.04 to 24.04

Reply:
all removed

Reply:
IMO, we can keep `ubuntu-latest` and let it update whenever the runners upgrade the default ubuntu they use.